snapshot/git/gitdb/stream.go

```diff
@@ -0,0 +1,66 @@
+package gitdb
+
+import (
+	"fmt"
+
+	"github.com/scootdev/scoot/snapshot"
+)
+
+// A Stream is a sequence of SnapshotWithHistory's that updates via a
```

Comment:
We can punt but I've been saying all along that Stream doesn't seem like the most natural fit.